h2o-py/h2o/expr.py

```diff
@@ -121,24 +120,44 @@ def _eval_driver(self, top):
     # the timezone change, so cannot be lazy.
     #
     def _get_ast_str(self, top):
```

Comment:
The comment should be changed to reflect the change of computation not being recursive anymore.

I would also document the meaning of `top` argument as I couldn't find the documentation anywhere. Does it mean that the frame should be cached ?